core/offchain/src/testing.rs

```diff
@@ -241,4 +241,3 @@ impl offchain::Externalities for TestOffchainExt {
 		}
 	}
 }
-
```

Comment:
Add line

Reply:
The standard git commit hook complained about a blank line at EOF during a merge, but I can skip that check.